package.json

```diff
@@ -40,12 +40,15 @@
     "@atomist/automation-client": "1.3.0-master.20190115210858",
     "@atomist/sdm": "1.3.0-master.20190125092715",
     "@atomist/sdm-core": "1.3.0-master.20190124142143",
+    "@atomist/sdm-pack-spring": "^1.1.1-master.20190131212713",
```

Comment:
Why do we have a spring pack runtime dependency suddenly?

Reply:
That's a dev dependency for the test sdm I added to the pack: https://github.com/atomist/sdm-pack-docker/blob/ipcrm_multi_registry/test/atomist.config.ts